TWLight/users/templates/users/editor_detail_data.html

```diff
@@ -127,15 +127,20 @@
       {% else %}
           {% comment %}Translators: When viewing a user's profile (e.g. https://wikipedialibrary.wmflabs.org/users/ when logged in), this shows if the user does not qualify the technical requirements. {% endcomment %}
           <strong class="warning">{% trans "No" %}</strong>
-          <p class="block-notice">
+          {% if editor.ignore_wp_blocks %}
+            {% comment %}Translators: If a user has received an exemption against the 'no blocks' criterion, this is shown next to "No" on their user page and applications next to this criterion. {% endcomment %}
```

Comment:
making this generic so we can reuse it with other criteria exemptions.
```suggestion
            {% comment %}Translators: If a user has received an exemption against a criterion, this is shown next to "No" on their user page and applications next to this criterion. {% endcomment %}
```